code/game/objects/items/crayons.dm

```diff
@@ -147,6 +147,29 @@
 	colour = "#DA00FF"
 	colourName = "purple"
 
+/obj/item/toy/crayon/random/New()
+	icon_state = pick(list("crayonred", "crayonorange", "crayonyellow", "crayongreen", "crayonblue", "crayonpurple"))
+	switch(icon_state)

```

Comment:
White are the mime ones, so I didn't stick it in there.

RGB Crayons would be cool, I suspect someone would do something very cool with it IC art-wise.